docs/libcurl/opts/CURLOPT_FORM_ESCAPE_AS_MIME.3

```diff
@@ -0,0 +1,75 @@
+.\" **************************************************************************
+.\" *                                  _   _ ____  _
+.\" *  Project                     ___| | | |  _ \| |
+.\" *                             / __| | | | |_) | |
+.\" *                            | (__| |_| |  _ <| |___
+.\" *                             \___|\___/|_| \_\_____|
+.\" *
+.\" * Copyright (C) 1998 - 2021, Daniel Stenberg, <daniel@haxx.se>, et al.
+.\" *
+.\" * This software is licensed as described in the file COPYING, which
+.\" * you should have received as part of this distribution. The terms
+.\" * are also available at https://curl.se/docs/copyright.html.
+.\" *
+.\" * You may opt to use, copy, modify, merge, publish, distribute and/or sell
+.\" * copies of the Software, and permit persons to whom the Software is
+.\" * furnished to do so, under the terms of the COPYING file.
+.\" *
+.\" * This software is distributed on an "AS IS" basis, WITHOUT WARRANTY OF ANY
+.\" * KIND, either express or implied.
+.\" *
+.\" **************************************************************************
+.\"
+.TH CURLOPT_FORM_ESCAPE_AS_MIME 3 "14 Oct 2021" "libcurl 7.80.0" "curl_easy_setopt options"
+.SH NAME
+CURLOPT_FORM_ESCAPE_AS_MIME \- change form field/file names escaping
+.SH SYNOPSIS
+#include <curl/curl.h>
+
+CURLcode curl_easy_setopt(CURL *handle, CURLOPT_FORM_ESCAPE_AS_MIME, long onoff);
+.SH DESCRIPTION
+Set the \fIonoff\fP parameter to 1 to escape multipart form field and file
+names using the backslash-escaping algorithm defined for mime mail rather than
+the percent-escaping used in URI.
```

Comment:
I think this description should refrain from associating the different escape mechanisms with mail vs URI since  both are and have been done over HTTP in HTTP request bodies a lot, so I think it mostly confuses the matter.

Let's just call them backslash-escaping and percent-encoding.

I think it could help to mention that browsers used to do backslash-escaping in the past but have over time transitioned to use percent-encoding.

Reply:
Will do.

Reply:
Done